code/game/objects/items/ego_weapons/_ego_weapon.dm

```diff
@@ -14,6 +14,8 @@
 							JUSTICE_ATTRIBUTE = 0
 							)
 
+	var/special = "This weapon has no special abilities."
```

Comment:
Variable name is ambiguous, I'd suggest changing it to something like "special_description".